task_processing/plugins/kubernetes/task_config.py

```diff
@@ -118,6 +121,27 @@ def _valid_volumes(volumes: Sequence["DockerVolume"]) -> Tuple[bool, Optional[st
     return (True, None)
 
 
+def _valid_empty_volumes(volumes: Sequence[EmptyVolume]) -> Tuple[bool, Optional[str]]:
+    for volume in volumes:
+        if not set(volume.keys()) <= VALID_EMPTY_VOLUME_KEYS:
+            return (
+                False,
+                f'Invalid empty volume format, must only contain following keys: '
+                f'{VALID_EMPTY_VOLUME_KEYS}, got: {volume.keys()}'
+            )
+        if 'container_path' not in set(volume.keys()):
```

Comment:
i think this can be 
```suggestion
        if 'container_path' not in volume:
```

`in` on a dict will automatically call .keys() and .keys provides a dict view (which is set-like) so we don't need to turn this into a set or call keys()

Reply:
I still want to prove that its strictly a subset of valid empty volume keys since if i only do 'container_path' in sneaky other keys can sneak in.